proto/src/determined/api/v1/api.proto

```diff
@@ -1668,4 +1669,84 @@ service Determined {
       tags: "Internal"
     };
   }
+
+  // Get a set of roles by id
```

Comment:
- Can this be clarified to be roles with the corresponding IDs (clarifying it is plural and is role.id and not a different object id)
- Nitpick, but there's a part of the generated TypeScript bindings which only picks up proto comments if they end with a period, so I'd request adding that and regenerating bindings